convert_to_TUM/convert_to_tum.py

```diff
@@ -0,0 +1,194 @@
+import copy
```

Comment:
For python files I suggest using linter in order to fix code style to pep8 for better readability (maybe even apply automatic changes) - for now flake8 shows some warnings

Reply:
fixed except too long lines